image_op/bin_movie_in_time.m

```diff
@@ -1,26 +1,81 @@
-function M_b = bin_movie_in_time(M, bin_factor)

```

Comment:
Note, basically the entire file has been rewritten. It may be easier to review the file by clicking the `View` button associated with this file, rather than try to read the line-by-line diff.